src/main/webapp/dashboards/patient.jsp

```diff
@@ -0,0 +1,19 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: tomgo
+  Date: 11/12/2020
+  Time: 02:11
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8"%>
+<html>
+<head>
+    <title>Patient Dashboard</title>
```

Comment:
If the CSS PR gets merged in first we'll need to add the updates to these new JSP headers.